Dockerfile_k8s_sshjump

```diff
@@ -0,0 +1,24 @@
+FROM continuumio/miniconda3:22.11.1
+
+RUN apt update -y && \
+    apt install sudo vim openssh-server -y && \
+    rm -rf /var/lib/apt/lists/*
+
+# Setup SSH and generate hostkeys
+RUN mkdir -p /var/run/sshd && \
+    sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config && \
+    sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd && \
+    cd /etc/ssh/ && \
+    ssh-keygen -A
+
+RUN useradd -m -s /bin/bash sky && \
+    echo "sky ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers && \
+    echo 'Defaults        secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"' > /etc/sudoers.d/sky
+
+# Switch to sky user
+USER sky
+
+# Add /home/sky/.local/bin/ to PATH
+RUN echo 'export PATH="$PATH:$HOME/.local/bin"' >> ~/.bashrc
+
+WORKDIR /home/sky

```

Comment:
Curious - what do you think about reusing the image generated from Dockerfile_k8s (`us-central1-docker.pkg.dev/skypilot-375900/skypilotk8s/skypilot:latest`) instead of creating a new one?

Pros: Reduces image maintenance burden for us, may be faster to pull since cluster would likely already have the main `skypilot:latest` image for running tasks
Cons: Dockerfile_k8s is quite bloated because it installs SkyPilot and its dependencies.

Might be worth looking at image sizes for both before taking a call.

Reply:
@romilbhardwaj , done.